modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -405,16 +405,368 @@ Client authentication and server authentication for the BasicAuth extension are
+<4> File name to save the CPU profile to. The profiling starts when the Collector starts and is saved to the file when the Collector is terminated.
+
+[id="healthcheck-extension_{context}"]
+==== Health Check extension
+
+The Health Check extension enables an HTTP url that can be probed to check the status of the OpenTelemetry Collector. This extension can be used as a liveness and/or readiness probe on OpenShift.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+
+.OpenTelemetry Collector custom resource with configured Health Check extension
+[source,yaml]
+----
+  config: |
+    extensions:
+      health_check:
+        endpoint: "0.0.0.0:13133" <1>
+        tls: <2>
+          ca_file: "/path/to/ca.crt"
+          cert_file: "/path/to/cert.crt"
+          key_file: "/path/to/key.key"
+        path: "/health/status" <3>
+        check_collector_pipeline: <4>
+          enabled: true <5>
+          interval: "5m" <6>
+          exporter_failure_threshold: 5 <7>
+
+    receivers:
+      otlp:
+        protocols:
+          http:
+
+    exporters:
+      otlp:
+
+    service:
+      extensions: [health_check]
+      pipelines:
+        traces:
+          receivers: [otlp]
+          exporters: [otlp]
+----
+<1> Address to publish the health check status. Default is `0.0.0.0:13133`.
+<2> The TLS server side configuration. Defines paths to TLS certificates. If omitted, TLS is disabled.
+<3> Specifies the path to be configured for the health check server. Default is `/`.
+<4> Settings of collector pipeline health check.
+<5> Enableds the collector pipeline check. default is `false`.
+<6> Time interval to check the number of failures. Default is `5m`.
+<7> The failure number threshold to mark containers as healthy. Default is `5`.
+
+
+[id="balast-extension_{context}"]
+==== Balast extension
+
+Memory Ballast extension enables applications to configure memory ballast for the process.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+
+.OpenTelemetry Collector custom resource with configured Balast extension
+[source,yaml]
+----
+  config: |
+    extensions:
+      memory_ballast:
+        size_mib: 64 <1>
+        size_in_percentage: 20 <2>
+
+    receivers:
+      otlp:
+        protocols:
+          http:
+
+    exporters:
+      otlp:
+
+    service:
+      extensions: [memory_ballast]
+      pipelines:
+        traces:
+          receivers: [otlp]
+          exporters: [otlp]
+----
+<1> Is the memory ballast size, in MiB. Takes higher priority than `size_in_percentage` if both are specified at the same time.
+<2> Sets the memory ballast based on the total memory in percentage, value range is 1-100. It is supported in both containerized(eg, docker, k8s) and physical host environments.
+
+
+[id="zpages-extension_{context}"]
+==== zPages extension
```

Comment:
The zPages extension should be as well documented in the troubleshooting page

Reply:
Do we already have a troubleshooting page for distr?

Reply:
https://docs.openshift.com/container-platform/4.13/distr_tracing/distr_tracing_otel/distr-tracing-otel-using.html

Reply:
https://docs.openshift.com/container-platform/4.13/distr_tracing/distr_tracing_otel/distr-tracing-otel-troubleshooting.html